custom/ilsgateway/migrations/0006_auto_20160211_1504.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
i generally prefer to give migrations a more descriptive name, for example 0006_add_<fieldname>